cmd/controller/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 			ctx,
 			op.Manager,
 			op.Config,
+			op.EC2API,
```

Comment:
This seems odd -- I wouldn't expect us to pass the raw API through here -- controllers should be reasoning in terms of providers, not APIs